tests/integration/utils2.go

```diff
@@ -1252,9 +1251,7 @@ func createDocViaColSave(
 
 func makeContextForDocCreate(ctx context.Context, action *CreateDoc) context.Context {
```

Comment:
todo: Sorry, this might be a problem with doc level encryption too that I missed, but by only testing via the context params it means that the GQL params are untested.  

I think it would be much better if we rely on context for the `collection-save` and `collection-named` mutation types, but the `gql` mutation type should really be using the new gql input params instead of the context (ideally it should test with various combinations, but that's probably not worth doing atm).

Reply:
done

Reply:
Quick sanity check, as I can't spot where this is done in the PR - did you push this change? How did you do this?

Reply:
it's right here https://github.com/sourcenetwork/defradb/pull/2817/commits/c38bca4cb18a19d48ef42be5ec81180ca1e8aaef#diff-13c3c742bdb7dd9a68f413104c129a63c831dab4e2b4d484be82115ab20812fcR1341

Reply:
I see my mistake - I actually misread originally - you were already using the GQL params (I didn't read closely enough and thought you were constructing a context key).  Thanks for the clarification :)